js/components/TagModal/TagModal.js

```diff
@@ -0,0 +1,41 @@
+import React, { Component } from 'react';
+import { View, Text, TouchableOpacity, Modal } from 'react-native';
+import styles from './styles';
+import PropTypes from 'prop-types';
+
+const TagModal = ({ open, close, title, description, backgroundColor }) => {
```

Comment:
you do not need open and close both as props. Just have one bool to help you. If close is a method, name it something else like closeModal.